docs/pages/typography.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: Typography
+name: Typography
+path: /typography
+---
+
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+} from '@deque/cauldron-react'
+
+Documentation and examples for Cauldron typography, including global settings, headings, body text, lists, and more.
+
+### Font
+
+Font tokens tie an intention to a font family.
+
+<Table>
+  <TableHead>
+    <TableRow>
+      <TableHeader scope="col">Name</TableHeader>
+      <TableHeader scope="col">Value</TableHeader>
+      <TableHeader scope="col">Description</TableHeader>
+    </TableRow>
+  </TableHead>
+  <TableBody>
+    <TableRow>
+      <TableCell>Base</TableCell>
+      <TableCell>Roboto', Helvetica, Arial, sans-serif</TableCell>
```

Comment:
Can we pull this value from the css var? There's a risk it will drift if we don't.